### ISSUE TITLE
Nordlond Bestiary Character Sheet seems to have gone missing with 1.06 release

### ISSUE THREAD

**Issue by Farmeroz** [crnormand/gurps#1433]
Updated from 1.05 to 1.06 of the Nordlond Bestiary and now the Nordlond character sheet isn't available?

**Comment by mjeffw** [comment]
Umm, I didn't see that when I tested, but I am also using an unreleased version of GGA. Let's me see if I can get GGA released today and you can test again.

**Comment by mjeffw** [comment]
@Farmeroz What exactly did you mean? Do the Nordlond Bestiary entries no longer open with the Bestiary actor sheet? Or are you trying to do something else?

**Comment by Farmeroz** [comment]
@mjeffw sorry for the delay - time zones!  Exactly that, they no longer open with the Bestiary actor sheet and if you look for it in the list of sheets it's not there.  Previous version, it was all good.  They came up in the Bestiary actor sheet and you could even change sheets, even edit for the actor, and then pop back to the bestiary.  Did the 1.06 update and now they're not not showing and the sheet isn't in the list.

**Comment by mjeffw** [comment]
Are there any errors in the developers console?

**Comment by Farmeroz** [comment]
Just updated to 1.6.2 and also installed on a different world.  Same issue.  Note, my installation is hosted on The Forge.

Errors:

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'addModifier')
[Detected 2 packages: system:gurps, lib-wrapper]
    at ranges.js:131:51
    at Array.forEach (<anonymous>)
    at GURPSRange._buildModifiers (ranges.js:130:17)
    at new GURPSRange (ranges.js:40:10)
    at gurps.js:1815:23
    at Function._call (foundry.js:294:14)
    at Function.callAll (foundry.js:253:12)
    at Game.initialize (foundry.js:5419:11)
    at Wrapper.🎁call_wrapped [as call_wrapped] (libWrapper-wrapper.js:511:22)
    at Game.🎁libWrapperInit (libWrapper-api.js:793:11)
target-enhancements.js:232 
        
       Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'clear')
[Detected 1 package: target-enhancements]
    at hoverTokenEventHandler (target-enhancements.js:232:22)
    at Function._call (foundry.js:294:14)
    at Function.callAll (foundry.js:253:12)
    at GurpsToken._onHoverIn (foundry.js:21895:11)
    at GurpsToken._onHoverIn (foundry.js:39028:18)
    at MouseInteractionManager.callback (foundry.js:24252:45)
    at MouseInteractionManager._handleMouseOver (foundry.js:24369:17)
    at GurpsToken.a.emit (index.js:181:35)
    at r.dispatchEvent (interaction.min.js:8:12018)
    at r.processPointerMove (interaction.min.js:8:16963)
    at t.recursiveFindHit (interaction.min.js:8:3973)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.findHit (interaction.min.js:8:4032)
    at r.processInteractive (interaction.min.js:8:12624)
    at r.onPointerMove (interaction.min.js:8:16395)
target-enhancements.js:408 
        
       Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'clear')
[Detected 3 packages: target-enhancements, zoom-pan-options, lib-wrapper]
    at controlTokenEventHandler (target-enhancements.js:408:28)
    at Function._call (foundry.js:294:14)
    at Function.callAll (foundry.js:253:12)
    at GurpsToken.control (foundry.js:21708:11)
    at GurpsToken._onClickLeft (foundry.js:21931:19)
    at GurpsToken._onClickLeft (foundry.js:39049:11)
    at MouseInteractionManager.callback (foundry.js:24252:45)
    at MouseInteractionManager._handleClickLeft (foundry.js:24439:10)
    at MouseInteractionManager._handleMouseDown (foundry.js:24419:22)
    at Wrapper.🎁call_wrapped [as call_wrapped] (libWrapper-wrapper.js:511:22)
    at MouseInteractionManager._handleMouseDown_Wrapper (zoom-pan-options.js:213:53)
    at MouseInteractionManager.🎁MouseInteractionManager.prototype._handleMouseDown#zoom-pan-options (zoom-pan-options.js:430:49)
    at Wrapper.🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:620:16)
    at MouseInteractionManager.🎁MouseInteractionManager.prototype._handleMouseDown#0 (libWrapper-wrapper.js:189:20)
    at GurpsToken.a.emit (index.js:181:35)
    at r.dispatchEvent (interaction.min.js:8:12018)
    at r.processPointerDown (interaction.min.js:8:13997)
    at t.recursiveFindHit (interaction.min.js:8:3973)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.findHit (interaction.min.js:8:4032)
    at r.processInteractive (interaction.min.js:8:12624)
    at r.onPointerDown (interaction.min.js:8:13326)
target-enhancements.js:232 
        
       Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'clear')
[Detected 1 package: target-enhancements]
    at hoverTokenEventHandler (target-enhancements.js:232:22)
    at Function._call (foundry.js:294:14)
    at Function.callAll (foundry.js:253:12)
    at GurpsToken._onHoverOut (foundry.js:21912:11)
    at GurpsToken._onHoverOut (foundry.js:39037:18)
    at MouseInteractionManager.callback (foundry.js:24252:45)
    at MouseInteractionManager._handleMouseOut (foundry.js:24391:17)
    at GurpsToken.a.emit (index.js:181:35)
    at r.dispatchEvent (interaction.min.js:8:12018)
    at r.processPointerOverOut (interaction.min.js:8:18039)
    at r.processPointerMove (interaction.min.js:8:16823)
    at t.recursiveFindHit (interaction.min.js:8:3973)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.findHit (interaction.min.js:8:4032)
    at r.processInteractive (interaction.min.js:8:12624)
    at r.onPointerMove (interaction.min.js:8:16395)
target-enhancements.js:232 
        
       Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'clear')
[Detected 1 package: target-enhancements]
    at hoverTokenEventHandler (target-enhancements.js:232:22)
    at Function._call (foundry.js:294:14)
    at Function.callAll (foundry.js:253:12)
    at GurpsToken._onHoverIn (foundry.js:21895:11)
    at GurpsToken._onHoverIn (foundry.js:39028:18)
    at MouseInteractionManager.callback (foundry.js:24252:45)
    at MouseInteractionManager._handleMouseOver (foundry.js:24369:17)
    at GurpsToken.a.emit (index.js:181:35)
    at r.dispatchEvent (interaction.min.js:8:12018)
    at r.processInteractive (interaction.min.js:8:12884)
    at r.onPointerMove (interaction.min.js:8:16395)
target-enhancements.js:232 
        
       Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'clear')
[Detected 1 package: target-enhancements]
    at hoverTokenEventHandler (target-enhancements.js:232:22)
    at Function._call (foundry.js:294:14)
    at Function.callAll (foundry.js:253:12)
    at GurpsToken._onHoverOut (foundry.js:21912:11)
    at GurpsToken._onHoverOut (foundry.js:39037:18)
    at MouseInteractionManager.callback (foundry.js:24252:45)
    at MouseInteractionManager._handleMouseOut (foundry.js:24391:17)
    at GurpsToken.a.emit (index.js:181:35)
    at r.dispatchEvent (interaction.min.js:8:12018)
    at r.processPointerOverOut (interaction.min.js:8:18039)
    at r.processPointerMove (interaction.min.js:8:16823)
    at t.recursiveFindHit (interaction.min.js:8:3973)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.findHit (interaction.min.js:8:4032)
    at r.processInteractive (interaction.min.js:8:12624)
    at r.onPointerMove (interaction.min.js:8:16395)
target-enhancements.js:232 
        
       Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'clear')
[Detected 1 package: target-enhancements]
    at hoverTokenEventHandler (target-enhancements.js:232:22)
    at Function._call (foundry.js:294:14)
    at Function.callAll (foundry.js:253:12)
    at GurpsToken._onHoverIn (foundry.js:21895:11)
    at GurpsToken._onHoverIn (foundry.js:39028:18)
    at MouseInteractionManager.callback (foundry.js:24252:45)
    at MouseInteractionManager._handleMouseOver (foundry.js:24369:17)
    at GurpsToken.a.emit (index.js:181:35)
    at r.dispatchEvent (interaction.min.js:8:12018)
    at r.processInteractive (interaction.min.js:8:12884)
    at r.onPointerMove (interaction.min.js:8:16395)
target-enhancements.js:408 
        
       Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'clear')
[Detected 1 package: target-enhancements]
    at controlTokenEventHandler (target-enhancements.js:408:28)
    at Function._call (foundry.js:294:14)
    at Function.callAll (foundry.js:253:12)
    at GurpsToken.release (foundry.js:21740:11)
    at GurpsToken._onDelete (foundry.js:21659:10)
    at GurpsToken._onDelete (foundry.js:38966:18)
    at TokenDocument._onDelete (foundry.js:9849:38)
    at ClientDatabaseBackend.callback (foundry.js:10488:11)
    at foundry.js:10471:43
    at Array.map (<anonymous>)
    at ClientDatabaseBackend._handleDeleteEmbeddedDocuments (foundry.js:10471:33)
    at ClientDatabaseBackend._deleteEmbeddedDocuments (foundry.js:10375:17)
    at async Function.deleteDocuments (document.mjs:410:21)
target-enhancements.js:232 
        
       Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'clear')
[Detected 1 package: target-enhancements]
    at hoverTokenEventHandler (target-enhancements.js:232:22)
    at Function._call (foundry.js:294:14)
    at Function.callAll (foundry.js:253:12)
    at GurpsToken._onHoverIn (foundry.js:21895:11)
    at GurpsToken._onHoverIn (foundry.js:39028:18)
    at MouseInteractionManager.callback (foundry.js:24252:45)
    at MouseInteractionManager._handleMouseOver (foundry.js:24369:17)
    at GurpsToken.a.emit (index.js:181:35)
    at r.dispatchEvent (interaction.min.js:8:12018)
    at r.processPointerMove (interaction.min.js:8:16963)
    at t.recursiveFindHit (interaction.min.js:8:3973)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.findHit (interaction.min.js:8:4032)
    at r.processInteractive (interaction.min.js:8:12624)
    at r.onPointerMove (interaction.min.js:8:16395)
target-enhancements.js:408 
        
       Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'clear')
[Detected 3 packages: target-enhancements, zoom-pan-options, lib-wrapper]
    at controlTokenEventHandler (target-enhancements.js:408:28)
    at Function._call (foundry.js:294:14)
    at Function.callAll (foundry.js:253:12)
    at GurpsToken.control (foundry.js:21708:11)
    at GurpsToken._onClickLeft (foundry.js:21931:19)
    at GurpsToken._onClickLeft (foundry.js:39049:11)
    at MouseInteractionManager.callback (foundry.js:24252:45)
    at MouseInteractionManager._handleClickLeft (foundry.js:24439:10)
    at MouseInteractionManager._handleMouseDown (foundry.js:24419:22)
    at Wrapper.🎁call_wrapped [as call_wrapped] (libWrapper-wrapper.js:511:22)
    at MouseInteractionManager._handleMouseDown_Wrapper (zoom-pan-options.js:213:53)
    at MouseInteractionManager.🎁MouseInteractionManager.prototype._handleMouseDown#zoom-pan-options (zoom-pan-options.js:430:49)
    at Wrapper.🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:620:16)
    at MouseInteractionManager.🎁MouseInteractionManager.prototype._handleMouseDown#0 (libWrapper-wrapper.js:189:20)
    at GurpsToken.a.emit (index.js:181:35)
    at r.dispatchEvent (interaction.min.js:8:12018)
    at r.processPointerDown (interaction.min.js:8:13997)
    at t.recursiveFindHit (interaction.min.js:8:3973)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.findHit (interaction.min.js:8:4032)
    at r.processInteractive (interaction.min.js:8:12624)
    at r.onPointerDown (interaction.min.js:8:13326)
target-enhancements.js:232 
        
       Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'clear')
[Detected 1 package: target-enhancements]
    at hoverTokenEventHandler (target-enhancements.js:232:22)
    at Function._call (foundry.js:294:14)
    at Function.callAll (foundry.js:253:12)
    at GurpsToken._onHoverOut (foundry.js:21912:11)
    at GurpsToken._onHoverOut (foundry.js:39037:18)
    at MouseInteractionManager.callback (foundry.js:24252:45)
    at MouseInteractionManager._handleMouseOut (foundry.js:24391:17)
    at GurpsToken.a.emit (index.js:181:35)
    at r.dispatchEvent (interaction.min.js:8:12018)
    at r.processPointerOverOut (interaction.min.js:8:18039)
    at r.processPointerMove (interaction.min.js:8:16823)
    at t.recursiveFindHit (interaction.min.js:8:3973)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.recursiveFindHit (interaction.min.js:8:3757)
    at t.findHit (interaction.min.js:8:4032)
    at r.processInteractive (interaction.min.js:8:12624)
    at r.onPointerMove (interaction.min.js:8:16395)
```

**Comment by Farmeroz** [comment]
Warnings:

```
lib.js:110 
        
       chat-portrait | Init Hooks processing
warn @ lib.js:110
lib.js:44 
        
       foundryvtt-arms-reach | Init Hooks processing
warn @ lib.js:44
socketlib.js:81 
        
       socketlib | Function 'recreateToken' is already registered for 'system.gurps'. Ignoring registration request.
register @ socketlib.js:81
socketlib.js:81 
        
       socketlib | Function 'createToken' is already registered for 'system.gurps'. Ignoring registration request.
register @ socketlib.js:81
socketlib.js:81 
        
       socketlib | Function 'deleteToken' is already registered for 'system.gurps'. Ignoring registration request.
register @ socketlib.js:81
socketlib.js:81 
        
       socketlib | Function 'renameToken' is already registered for 'system.gurps'. Ignoring registration request.
register @ socketlib.js:81
socketlib.js:81 
        
       socketlib | Function 'setTokenFlag' is already registered for 'system.gurps'. Ignoring registration request.
register @ socketlib.js:81
socketlib.js:81 
        
       socketlib | Function 'setFlag' is already registered for 'system.gurps'. Ignoring registration request.
register @ socketlib.js:81
socketlib.js:81 
        
       socketlib | Function 'unsetFlag' is already registered for 'system.gurps'. Ignoring registration request.
register @ socketlib.js:81
socketlib.js:81 
        
       socketlib | Function 'deleteUuid' is already registered for 'system.gurps'. Ignoring registration request.
register @ socketlib.js:81
foundry.js:5702 
        
       Font Family "Helvetica" could not be loaded and was removed
(anonymous) @ foundry.js:5702
Error with Permissions-Policy header: Origin trial controlled feature not enabled: 'interest-cohort'.
```

**Comment by Farmeroz** [comment]
And no Nordlond sheet visible:

![image](https://user-images.githubusercontent.com/89975749/170595275-6ced1d40-7169-44a6-827d-8dbe0006d5d2.png)

**Comment by Farmeroz** [comment]
This is resolved with release of GGA 0.14.1 - thank you!
